react/week4/gitusers/src/components/SearchUser.js

```diff
@@ -0,0 +1,22 @@
+import React, {useContext} from 'react'
```

Comment:
Please use Prettier and format files https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode